docs/source/supergraphs.md

```diff
@@ -74,3 +74,15 @@ rover supergraph compose --config ./supergraph.yaml > prod-schema.graphql
 ```
 
 > For more on passing values via `stdout`, see [Using `stdout`](./conventions#using-stdout).
+
+#### Gateway compatibility
+
+`rover supergraph compose` utilizes the composition function published in the [harmonizer]() package under the hood. Harmonizer is still under rapid development, and some breaking changes are to be expected. For now, newer versions of Rover emit supergraph schemas that are incompatible with older versions of [`@apollo/gateway`](https://www.npmjs.com/package/@apollo/gateway).
+
+The following table should help you understand the compatible versions of these two pieces of software.
+
+
+|Rover version|Gateway version|
+|---|---|
+|<= v0.2|<= v0.38|
+|>= v0.3|>= v0.39|
```

Comment:
Is there an existing or hypothetical future rover version 'x' that ought to be accounted for in the range `0.2<x<0.3`?

Reply:
v0.2.1 is out today and also would need an older version - maybe i'll just add the `.x` in the table